core/include/mm/tee_mmu_types.h

```diff
@@ -47,20 +47,14 @@
 #define	TEE_MATTR_URWX		(TEE_MATTR_URW | TEE_MATTR_UX)
 
 #define TEE_MATTR_GLOBAL	(1 << 10)
-#define TEE_MATTR_I_NONCACHE	 0
-#define TEE_MATTR_I_WRITE_THR	(1 << 11)
-#define TEE_MATTR_I_WRITE_BACK	(1 << 12)
-#define TEE_MATTR_O_NONCACHE	 0
-#define TEE_MATTR_O_WRITE_THR	(1 << 13)
-#define TEE_MATTR_O_WRITE_BACK	(1 << 14)
+#define	TEE_MATTR_SECURE	(1 << 11)
 
-#define TEE_MATTR_NONCACHE	 0
-#define TEE_MATTR_CACHE_DEFAULT	(TEE_MATTR_I_WRITE_BACK | \
-				 TEE_MATTR_O_WRITE_BACK)
+#define TEE_MATTR_CACHE_MASK	0x7
```

Comment:
For the ARM architecture we can support up to 3 bits for this field. Even if we're currently only using one I think it's good to have some spare bits just in case.